### ISSUE TITLE
False positive for rule no-yield-in-race when Array.map() is used inside race()

### ISSUE THREAD

**Issue by apaatsio** [pke/eslint-plugin-redux-saga#47]
The following code triggers the `no-yield-in-race` rule (*"race must have on object hash or array as the only argument"*) even though the argument to `race()` is an array and thus valid.

```javascript
function* foo() {
  const actions = ["LOGIN", "LOGOUT"]
  yield race(actions.map(take))
}
```
